### PR TITLE
Bugfix issue #69 -  Backport on 0.9

### PR DESCRIPTION
**Pull request by julien-gonzalez**
Backport of the bugfix for issue #69 on the branch OWSI-Core-0.9 for project purpose.

### CODE REVIEW

File: owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/main/java/fr/openwide/core/wicket/more/util/visit/VisitFilters.java [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+				return false;
+			}
+			Component parent = component.getParent();
+			return component.isVisibleInHierarchy()
+					|| (parent == null || parent.isVisibleInHierarchy()) && component.getOutputMarkupPlaceholderTag();
+		}
+
+		@Override
+		public boolean visitChildren(Object object) {
+			return hasRenderedMarkup(object);
```
Comment (lalmeras) [inline owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/main/java/fr/openwide/core/wicket/more/util/visit/VisitFilters.java:222]: I'm not sure that renderedComponents is the better choice as it visits the component's children, so I think that you'll replace too much element in you AjaxRequestTarget ?

Comment (lalmeras) [reply owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/main/java/fr/openwide/core/wicket/more/util/visit/VisitFilters.java:222]: No, it's fine ; we don't use a visitor ; we just use visitObject method, so there is no hierarchy traversal. Fine.

**Comment by lalmeras** [comment]
Can you rework you commit's comment:

- to limit first line to 80 characters so that it can be kept on a unique line in history
- to add explicitly (if this is the case) that VisitFilters.java is just a backported class. Also, let me know if significant changes were needed to adapt this class to 0.9, so that we can write it down on the comment.

**Review by lalmeras — CHANGES_REQUESTED** [review]
Can you rework your commit's comment:

- to limit first line to 80 characters so that it can be kept on a unique line in history
- to add explicitly (if this is the case) that VisitFilters.java is just a backported class. Also, let me know if significant changes were needed to adapt this class to 0.9, so that we can write it down on the comment.

**Comment by julien-gonzalez** [comment]
Requested adjustments on commit message are done.

**Review by lalmeras — APPROVED** [review]

**Pull request merged**